app/search/buckets.py

```diff
@@ -44,7 +44,6 @@ class BucketKeys(StrEnum):
     """
 
     TNA = "tna"
-    DIGITISED = "digitised"
```

Comment:
`DIGITISED = "digitised"` can be retained since its valid for online records and the change is specified for front end.